whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/SearchResult.java

```diff
@@ -14,7 +14,9 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "entity", propOrder = {
         "entityResults",
-        "domainResults"
+        "domainResults",
+        "autnumResults",
+        "networkResults"
 })
```

Comment:
why autnumResults and networkResults are added? are they being used somewhere through searchResult bean?

Reply:
I added it in the beginning because I thought that I would need to use seachResult and I totally forgot to delete them